pkg/ccl/streamingccl/streamproducer/producer_job_test.go

```diff
@@ -116,7 +116,7 @@ func TestStreamReplicationProducerJob(t *testing.T) {
 	sql.Exec(t, "SET CLUSTER SETTING stream_replication.stream_liveness_track_frequency = '1ms'")
 	registry := source.JobRegistry().(*jobs.Registry)
 	ptp := source.DistSQLServer().(*distsql.ServerImpl).ServerConfig.ProtectedTimestampProvider
-	timeout, username := 1*time.Second, username.MakeSQLUsernameFromPreNormalizedString("user")
+	timeout, usr := 1*time.Second, username.MakeSQLUsernameFromPreNormalizedString("user")
```

Comment:
[nit] I don't see any functional changes in this file.  Style cleanups are fine, but I think we try to prefer doing them when we making other changes to the code in question or as separate commits.  I'm fine with it here though since this code is in flux.